src/config/dynamic-theme-fixes.config

```diff
@@ -185,6 +185,15 @@ CSS
 
 ================================
 
+*.galaxus.de
+
+CSS
+.sc-fdf7076e-0 {
```

Comment:
maybe change this selector to: `#sidebarMenu div`
This class looks like it can be randomly generated name. 😉